libs/shared/contentful/src/lib/contentful.client.ts

```diff
@@ -204,9 +204,11 @@ export class ContentfulClient {
     const cacheTTL =
       this.contentfulClientConfig.memCacheTTL || DEFAULT_MEM_CACHE_TTL * 1000;
 
-    setInterval(() => {
+    const clear = setInterval(() => {
       this.locales = [];
       this.graphqlMemCache = {};
+      clearInterval(clear);
     }, cacheTTL);
+    clear.unref();
```

Comment:
These changes should be reverted. With these changes the function in `setInterval` doesn't get executed.

Was this change made to resolve the open handles during testing? If so, were [Jest timer mocks](https://jestjs.io/docs/timer-mocks) considered instead?